Tests/test_PDB.py

```diff
@@ -19,6 +19,7 @@
 import unittest
 import warnings
 from Bio._py3k import StringIO
+from Bio.PDB.PDBParser import PDBParser
```

Comment:
This import needs to be after the check for if numpy is installed (so that ``run_tests.py`` can skip the whole test if NumPy is missing).